k8s/env-cluster.sh

```diff
@@ -18,3 +15,7 @@ fi
 # Machine names
 ENV_INFRASTRUCTURE_FILE="$CLUSTER_CONFIG_DIR/env-infrastructure.sh"
 . "$ENV_INFRASTRUCTURE_FILE"
```

Comment:
No need to check if this file (o its parent folder) actually exist before attempting to execute the one?